src/Storefront/Resources/snippet/en_GB/storefront.en-GB.json

```diff
@@ -1,480 +1,480 @@
 {
-    "general": {
```

Comment:
Indents again

Reply:
just fixed indents from 4 to 2. no text changes